### PR TITLE
Update Overview of Debugging Tools for 4.3.

### PR DESCRIPTION
**Pull request by chrisl8**
- I have updated the Debug drop-down menu image to how it appears in 4.3/Master.
- I have also added an image of the new Run Customize Instances.,.. dialogue.
- I have updated the description of the entries in the Debug drop-down to match the tool-tips for those features more closely.
- I have added documentation and tips and warnings for using the Run Customize Instances.,.. dialogue.

This is intended to document the changes made in this PR:
https://github.com/godotengine/godot/pull/88685

### CODE REVIEW

File: tutorials/scripting/debug/overview_of_debugging_tools.rst [hunk 1]
```diff
@@ -52,26 +53,110 @@ Also, on Android, the game is deployed using the USB cable to speed up deploymen
...
 
-Navigation meshes and polygons will be visible on the running game.
+When this option is enabled, navigation meshes and polygons will be visible in the running project.
 
-Sync Scene Changes
-++++++++++++++++++
+Visible Avoidance
++++++++++++++++++
+
+When this option is enabled, avoidance objects shapes, radius and velocities will be visible in the running project.
```
Comment (AThousandShips) [inline tutorials/scripting/debug/overview_of_debugging_tools.rst:71]:
```suggestion
When this option is enabled, avoidance object shapes, radiuses, and velocities will be visible in the running project.
```

File: tutorials/scripting/debug/overview_of_debugging_tools.rst [hunk 2]
```diff
@@ -52,26 +53,110 @@ Also, on Android, the game is deployed using the USB cable to speed up deploymen
...
+Visible Paths
++++++++++++++
+
+When this option is enabled, curve resources used by path nodes will be visible in the running project.
 
 Visible Navigation
 ++++++++++++++++++
 
-Navigation meshes and polygons will be visible on the running game.
+When this option is enabled, navigation meshes and polygons will be visible in the running project.
```
Comment (AThousandShips) [inline tutorials/scripting/debug/overview_of_debugging_tools.rst:66]:
```suggestion
When this option is enabled, navigation meshes, and polygons will be visible in the running project.
```

File: tutorials/scripting/debug/overview_of_debugging_tools.rst
Comment (AThousandShips) [inline tutorials/scripting/debug/overview_of_debugging_tools.rst:1]: Please break up some of your long lines, use the existing lines as a reference, too long lines are hard to read in the source

File: tutorials/scripting/debug/overview_of_debugging_tools.rst (hunk 1 above)
Comment (chrisl8) [reply tutorials/scripting/debug/overview_of_debugging_tools.rst:71]: FYI: This is a direct copy of the tool-tip
https://github.com/godotengine/godot/blob/fe01776f05b1787b28b4a270d53037a3c25f4ca2/editor/plugins/debugger_editor_plugin.cpp#L83

I guess I could fix that too!

**Comment by chrisl8** [comment]
I've made all of the corrections. I amended my commit rather than making a new one so as to keep everything in one commit, while avoiding a squash, as I believe is the preferred Godot workflow.

Let me know if you need anything else here. Thanks.

**Review by AThousandShips — APPROVED** [review]
Looks good to me style wise, can't comment on the facts

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Merged. Thanks and congrats on your first merged contribution!

**Pull request merged**